src/Network/Network/help/Az.Network.md

```diff
@@ -132,7 +141,7 @@ Adds a VPN client root certificate.
 Approves a private endpoint connection.
 
 ### [Deny-AzPrivateEndpointConnection](Deny-AzPrivateEndpointConnection.md)
-Denies a private endpoint connection.
+denies a private endpoint connection.
```

Comment:
This change shouldn't happen

Reply:
This is the result from running the script. I didn't add it. I can manually remove it

Reply:
manually changed back